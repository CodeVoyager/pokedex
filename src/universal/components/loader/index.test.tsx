import { shallow } from 'enzyme';
import React from 'react';
import { Loader } from '.';

describe('Component::ButtonsContainer', () => {
  it('should render correctly', () => {
    expect(shallow(<Loader />)).toMatchSnapshot();
  });
});
