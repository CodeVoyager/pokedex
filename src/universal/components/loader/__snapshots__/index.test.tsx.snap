// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component::ButtonsContainer should render correctly 1`] = `
<div
  className="pokemon-loader"
>
  <div
    className="pokemon-loader-ring"
  >
    <div
      className="pokemon-loader-center"
    />
  </div>
</div>
`;
